### PR TITLE
tidb-in-kubernetes: improve content and wording of `deploy-tidb-from-k8s-kind` doc

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Update the link of Helm Client
- Update the content of `访问数据库和监控面板` and `部署 TiDB Operator`

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1855

### Which version does your change affect? 
dev, v3.0, v3.1

### CODE REVIEW

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 1]
```diff
@@ -108,15 +108,69 @@ kind 通过 Docker 容器模拟出一个本地的 Kubernetes 集群。kind 的
...
+        > **注意：**
+        >
+        > 如果代理建立成功，会打印类似输出：`Forwarding from 0.0.0.0:4000 -> 4000`。测试完成后按 `Ctrl + C` 停止代理并退出。
+
+    2. 然后，通过 MySQL 客户端访问 TiDB，打开一个**新**终端标签或者一个**新**终端窗口，执行下面的命令：
+
+        {{< copyable "shell-regular" >}}
+
+        ``` shell
+        mysql -h 127.0.0.1 -P 4000 -u root
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: If the kind is deployed on remote VM or host, we have to access the service with the VM or host IP, right?

Comment (juliezhang1112) [reply dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: PTAL Thanks!

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 2]
```diff
@@ -108,15 +108,73 @@ kind 通过 Docker 容器模拟出一个本地的 Kubernetes 集群。kind 的
...
+
+        ``` shell
+        kubectl port-forward svc/<release-name>-tidb 4000:4000 --namespace=<namespace>
+        ```
+
+        > **注意：**
+        >
+        > 如果代理建立成功，会打印类似输出：`Forwarding from 0.0.0.0:4000 -> 4000`。测试完成后按 `Ctrl + C` 停止代理并退出。
+
+    2. 然后，通过 MySQL 客户端访问 TiDB，打开一个**新**终端标签或者一个**新**终端窗口，执行下面的命令：
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
    2. 然后，通过 MySQL 客户端访问 TiDB，打开一个**新**终端标签或窗口，执行下面的命令：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 3]
```diff
@@ -108,15 +108,73 @@ kind 通过 Docker 容器模拟出一个本地的 Kubernetes 集群。kind 的
...
+
+        ``` shell
+        kubectl port-forward svc/<release-name>-grafana 3000:3000 --namespace=<namespace>
+        ```
+
+        > **注意：**
+        >
+        > 如果代理建立成功，会打印类似输出：`Forwarding from 0.0.0.0:3000 -> 3000`。测试完成后按 `Ctrl + C` 停止代理并退出。
+
+    2. 然后，在浏览器中打开 `http://localhost:3000` 访问 Grafana 监控面板：
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
    2. 然后，在浏览器中打开 <http://localhost:3000> 访问 Grafana 监控面板：
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 4]
```diff
@@ -108,15 +108,73 @@ kind 通过 Docker 容器模拟出一个本地的 Kubernetes 集群。kind 的
...
+    >
+    > 如果使用 kubectl 1.13 或者更高版本，可以在执行 `kubectl port-forward` 命令时添加 `--address 0.0.0.0` 选项，在 `0.0.0.0` 暴露端口而不是默认的 `127.0.0.1`：
+    >
+    > {{< copyable "shell-regular" >}}
+    >
+    > ```
+    > kubectl port-forward --address 0.0.0.0 -n tidb svc/<release-name>-grafana 3000:3000
+    > ```
+    >
+    > 然后，在浏览器中打开 `http://vmip:3000` 访问 Grafana 监控面板。
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
    > 然后，在浏览器中打开 <http://vmip:3000> 访问 Grafana 监控面板。
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 5]
```diff
@@ -167,11 +157,25 @@ kind 通过使用 Docker 容器作为集群节点模拟出一个本地的 Kubern
...
+    >
+    > 如果使用 kubectl 1.13 或者更高版本，可以在执行 `kubectl port-forward` 命令时添加 `--address 0.0.0.0` 选项，在 `0.0.0.0` 暴露端口而不是默认的 `127.0.0.1`：
+    >
+    > {{< copyable "shell-regular" >}}
+    >
+    > ```
+    > kubectl port-forward --address 0.0.0.0 -n tidb svc/<release-name>-grafana 3000:3000
+    > ```
+    >
+    > 然后，在浏览器中打开 <http://vmip:3000> 访问 Grafana 监控面板。
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: is it better to change `vmip` to `远程主机 IP`？

Comment (juliezhang1112) [reply dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]: Agree, `远程主机 IP` is clearer and more accurate.

**Comment by juliezhang1112** [comment]
PTAL again @DanielZhangQD 
Thanks~

**Pull request merged**